2024-12-23 11:56:47,874 - fastapi - ERROR - Form data requires "python-multipart" to be installed. 
You can install "python-multipart" with: 

pip install python-multipart

2024-12-23 11:57:35,650 - httpx - INFO - HTTP Request: POST https://viubkydrocuevnfffloo.supabase.co/auth/v1/token?grant_type=password "HTTP/2 200 OK"
2024-12-23 11:57:43,563 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x166f95ed0>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:57:51,347 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167ab9290>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:57:59,132 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167abbdd0>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:58:27,121 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a15110>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:58:34,906 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a15e90>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:58:42,690 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a16b10>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:58:58,257 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a21390>, 'Connection to localhost timed out. (connect timeout=30)')': /query-history
2024-12-23 11:59:06,038 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a21b10>, 'Connection to localhost timed out. (connect timeout=30)')': /query-history
2024-12-23 11:59:13,822 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a233d0>, 'Connection to localhost timed out. (connect timeout=30)')': /query-history
2024-12-23 11:59:28,151 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a2c0d0>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:59:35,932 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x162d4b510>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
2024-12-23 11:59:43,715 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x167a14e10>, 'Connection to localhost timed out. (connect timeout=30)')': /collections
